src/commands.rs

```diff
@@ -493,6 +494,16 @@ struct UnsquashArgs {
     interactive: bool,
 }
 
+/// Resolve a conflicted file with an external merge tool
+#[derive(clap::Args, Clone, Debug)]
+struct ResolveArgs {
+    #[arg(long, short, default_value = "@")]
+    revision: String,
+    // TODO: Make this optional, pick an arbitrary file with a conflict
```

Comment:
I think both hg and git will run the merge tool for each file. I'm not sure if we want to follow that and have a `--one` flag or if we want to default to resolving one path and have a `--all` flag to resolve all. We should probably allow multiple paths for this argument.

Also, some tools (like meld) support merging directories. I just tried meld on directories and it seems that it ignores the `--output` option then. Hmm...

Reply:
See also https://github.com/martinvonz/jj/pull/689#pullrequestreview-1176861557.

Reply:
This is an interesting question. If `jj` had a TUI for selecting one of the conflicts to resolve, that would be best. Without a TUI, we could use a text editor to have the user make a selection (a little awkwardly). I thought the interface suggested in this comment is a good compromise, but it has its problems.

Reply:
I'm fine with just resolving a single conflict to start with if that's easiest. We can add support for multiple conflicts later.